docs/ui/storage/fragments/web/s3-album.md

```diff
@@ -0,0 +1,149 @@
+## Installation
+
+<docs-filter framework="react">
+
+```
+yarn add aws-amplify @aws-amplify/ui-react
+```
+</docs-filter>
+<docs-filter framework="angular">
+
+```
+yarn add aws-amplify @aws-amplify/ui-angular
+```
+</docs-filter>
+<docs-filter framework="ionic">
+
+```
+yarn add aws-amplify @aws-amplify/ui-angular
+```
+</docs-filter>
+<docs-filter framework="vue">
+
+```
+yarn add aws-amplify @aws-amplify/ui-vue
+```
+</docs-filter>
```

Comment:
We should be able to migrate these lines to an `installation.md` inside of the fragments/web directory

Reply:
Agreed, since this is looking to be reused we can break it out and place this within the other documentation files